src/components/search/results.tsx

```diff
@@ -52,9 +52,10 @@ const SearchResults: React.FunctionComponent<ResultsProps> = ({
         }
     }, [pageNumber, filters, searchString, sortBy]);
 
-    useEffect(() => {
-        setCurrentPage(startPage);
-    }, [results]);
+    // TODO: Matt or Harika to confirm the necessity of the code below
```

Comment:
I don't see any negative impact of having this removed. the cause for results change is this https://github.com/mongodb/devcenter/blob/f4cb13ddfd6eb49ca5525219e71ac6cf5e9941fa/src/components/search/results.tsx#L67C39-L67C39.